File: Aula01/Principal.py
lista_numeros = []
lista2_numeros = []
lista_palavras = []

Util.popular_aleatorio_numeros(lista_numeros, 10, 100, 500)
Util.exibir_lista(lista_numeros)

lista2_numeros.extend(lista_numeros)

Util.popular_aleatorio_palavras(lista_palavras, 10, 6)
Util.exibir_lista(lista_palavras)

#----- Parte exercicio da aula -----

lista_numeros_arquivo = []
caminho = "numeros.txt"

Util.popular_numeros_de_arquivo(lista_numeros_arquivo, caminho)

lista_palavras_arquivo = []
caminho = "palavras.txt"

Util.popular_palavras_de_arquivo(lista_palavras_arquivo, caminho)

Util.exibir_lista(lista_numeros_arquivo)
Util.exibir_lista(lista_palavras_arquivo)

File: Aula01/Exercicio/Principal.java
import java.util.List;

public class Principal {
    public static void main(String[] args) {
        List<Integer> listaNumeros = new ArrayList<>();
        List<String> listaPalavras = new ArrayList<>();

        Util.popularAleatorioNumeros(listaNumeros, 10, 100, 500);
        Util.exibirListaNumeros(listaNumeros);

        System.out.println("\n----------\n");

        Util.popularAleatorioPalavras(listaPalavras, 10, 6);
        Util.exibirListaPalavras(listaPalavras);

        System.out.println("\n-------------\n");

        //---Parte exercicio da aula----

        List<Integer> listaNumerosArquivo = new ArrayList<>();

        String caminhoNumeros = "numeros.txt"; //path

        Util.PopularNumerosDeArquivo(listaNumerosArquivo, caminhoNumeros);
        Util.exibirListaNumeros(listaNumerosArquivo);

        System.out.println("\n---------------------\n");

        List<String> listaPalavrasArquivo = new ArrayList<>();

        String caminhoPalavras = "palavras.txt"; //path

        Util.popularPalavrasDeArquivo(listaPalavrasArquivo, caminhoPalavras);
        Util.exibirListaPalavras(listaPalavrasArquivo);
    }
}
